CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -89,6 +89,19 @@ protected function getTemplateUpdates() {
         'templates' => [
           ['name' => 'contribution_invoice_receipt', 'type' => 'html'],
         ]
+      ],
+      [
+        'version' => '5.9.alpha1',
```

Comment:
@mattwire can you change this to 5.10.alpha1 and squash your commits?

Reply:
Done - ready to merge?